homeassistant/helpers/script.py

```diff
@@ -775,6 +780,35 @@ async def async_run_sequence(iteration, extra_msg=""):
                 if self._stop.is_set():
                     break
 
+        elif CONF_FOR_EACH in repeat:
+            try:
+                items = template.render_complex(repeat[CONF_FOR_EACH], self._variables)
+            except (exceptions.TemplateError, ValueError) as ex:
+                self._log(
+                    "Error rendering %s repeat for each items template: %s",
+                    self._script.name,
+                    ex,
+                    level=logging.ERROR,
+                )
+                raise _AbortScript from ex
+
+            if not isinstance(items, list):
+                self._log(
+                    "For each items in %s repeat must be a list, got: %s",
+                    self._script.name,
+                    items,
+                    level=logging.ERROR,
+                )
+                raise _AbortScript("For each items must be a list")
```

Comment:
Let's rewrite this sentence or wrap `For each` in quotes so it's easier to understand (same with log above)

Reply:
Done!